service/src/enclave/api.rs

```diff
@@ -295,6 +302,7 @@ pub fn get_expired_key(enclave: &SgxEnclave) -> Option<(Vec<u8>, u32, u32)> {
 			&mut retval,
 			key.as_mut_ptr(),
 			AES_KEY_MAX_SIZE as u32,
+			nts_time,
```

Comment:
Here, you use nts_time to get expired keys from the outside of the enclave, what if this time is modified since this part of memory is not secured?